azerothcore/azerothcore-wotlk
# fix(Scripts/BlackwingLair): Chromaggus lever should stay open

Chromaggus lever door closes after reset, should stay open

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12743



## Changes Proposed:
-  Remove Chromaggus lever door as DOOR_TYPE_PASSAGE 

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12743

## SOURCE:


## Tests Performed:

- 
- 

## How to Test the Changes:

Go to chromaggus room
```
.go c 84648
```
1. Pull lever, see that door opens
2. Reset the boss by dying or turning on gm mode
3. Observe lever door stays open
4. Pull boss, see that lever door stays open

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
